src/schema/artwork/index.js

```diff
@@ -570,20 +518,20 @@ export const artworkFields = () => {
         return partnerLoader(partner.id).catch(() => null)
       },
     },
-    price: {
-      type: GraphQLString,
-    },
+    price: { type: GraphQLString },
+    domesticShipping: amount(
```

Comment:
Also was not sure if I'm supposed to use underscore_case or capitalCase for the name here. What are the current rules here?

Reply:
I think we're settled on 🐫 for newer things!

Reply:
I'm assuming the currency here is pretty much implied/assumed to all be USD?

As an FYI (now that I think about it I should extract this into the `money` schema), here's how we result in a displayable string for artist auction results: https://github.com/artsy/metaphysics/blob/1a055771148a142845d85226c9758c291898d579/src/schema/auction_result.js#L199-L217

It handles currency and the cents, and thus properly handles yen (not dividing cents by 100), correctly including the GBP symbol, etc.

But for now I think just using the `amount` helper should totally work!